### ISSUE TITLE
For help to solve the error of "Potential value was nan or inf. Aborting."

### ISSUE THREAD

**Issue by hanyecn** [potfit/potfit#64]
Hi,
 I'm using potfit to fit a meam potential function. The operating system is CentOS 7.0, the potfit version is: potfit-20190901 (d1ce015f). The code was compiled using Intel compiler with MKL math library. The options of mpi, meam, apot, meam and stress were enabled. I prepared three input files of test.config, test.meam and test.para. When I tried to run the fit, I got the error information of :"Potential value was nan or inf. Aborting." Please help me to find out what I can do to solve this problem. The configure file of test.config was obtained from a vasp OUTCAR using vasp2force, the potential file of test.meam was built using makeapot and the file of test.para was built using potfig_setup. Thanks a lot, and looking forward to hear from you soon.

**Comment by hanyecn** [comment]
I changed the fit functions from 6*lj,3*bjs,3*sqrt to 6*eopp_exp,3*poly_5,3*harmonic, now it's running. I have no idea about what is going on and why is this happening.

**Comment by hanyecn** [comment]
Now, I have figured it out the the meanings of the fit functions. For MEAM, 12 models were needed, 3 phi, 2 rho, 2 Fi, 3 fij and 2 gi. My question is which functions should I take for fij and gi. Thanks alot.

**Comment by dschopf** [comment]
There is no generic recommendation for the functions for a specific model. This is on the one hand highly dependant on the system you are trying to fit and on the other hand also on the final application of the potential. A potential for dynamic properties might work better with different functiond than a potential for static properties (like lattice constant, energies, etc.). 

The error "Potential value was nan or inf" indicates that there are some issues with the potential functions. Unfortunately these issues can be manyfold and we cannot give a much better error description. An example might be a function parameter which causes a "division by zero" error.

A way to fix this is by either chosing a different set of potential functions or trying to restart the fitting process with a different random number generator seed.

**Comment by hanyecn** [comment]
Thanks for your advice. I have changed potential functions, Now it's running. Is it possible to estimate the time for the running job? Could I check the convergence trend while the job is running? Thank you very much!

**Comment by dschopf** [comment]
> Is it possible to estimate the time for the running job?
No, this highly depends on the number of potential parameters, the number of atomic configurations and the number of CPU cores used (MPI).

> Could I check the convergence trend while the job is running?
Yes, for each optimization algorithm there is output to the console which will give you some numbers for the currently best potential. There is no way to determine a "convergence trend".

> Does potfit support writing MEAM potential in LAMMPS format now?
No, potfit does not support this.
